src/main/java/frc/robot/autonomous/paths/RightConeCubeHigh.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.autonomous.paths;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.autonomous.FollowPath;
+import frc.robot.commandGroups.PickUpCube;
+import frc.robot.commandGroups.UpperScoring;
+import frc.robot.subsystems.arm.ArmConstants;
+import frc.robot.subsystems.intake.ConstantsIntake;
+
+/**
+ * This class contains all the parts to the path RightConeCubeHigh.
+ *
+ * In this path the robot places a cone in the grid that is the furthest away from the feeder,
+ * goes to pick up a cube (the one furthest from the feeder)
+ * and returns to place it in the same grid.
+ */
+public class RightConeCubeHigh extends SequentialCommandGroup {
+    public RightConeCubeHigh() {
+        addCommands(
+                new UpperScoring(),
+                FollowPath.loadTrajectory(".pathplanner/RightConeCubeHigh blue 1"),
+                new PickUpCube(),
```

Comment:
Shouldn't you change the pipeline of the vision?

Reply:
That will be done when the operator decides which game piece to pick up